docs/DeveloperGuide.md

```diff
@@ -627,29 +722,27 @@ The predicate is then used to filter the list of person.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Appendix: Requirements**
+## **Appendix A: Requirements**
 
 ### Product scope
 
 **Target user profile**:
 
-A private 1-to-1 home tuition teacher in Singapore that:
+A private 1-to-1 home tuition teacher that:
 
 1. has a need to manage not more than 50 student contacts
-2. has a need to manage admin details of each student, namely, lesson fees and payment
+2. has a need to manage payment details of each student
 3. has a need to keep track of lesson details for each student
-4. teaches lessons anytime between 8am and 10pm (UTC+8), on any day of the week
+4. teaches lessons anytime between typical working hours of 8am and 10pm, on any day of the week
```

Comment:
I think specifying the timezone shouldn't be removed though

Reply:
I think this was our original intention but I think since the LocalDate/Time objects we were using alll uses System timing, they are not specific to UTC +8 anymore